web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java

```diff
@@ -627,7 +630,7 @@ public synchronized Map<String, Object[]> getErrors() {
 		 */
 		public void start() {
 			setUpdatesRequired(true);
-			thread.start();
+			OpenmrsThreadPoolHolder.threadExecutor.submit(r);
```

Comment:
This is basically the idiom for starting a background thread without any privileges.